### ISSUE TITLE
Error: Client network socket disconnected before secure TLS connection was established

### ISSUE THREAD

**Issue by Zack00e** [andresilvasantos/bitprophet#54]
![Screenshot_20211207_011539](https://user-images.githubusercontent.com/40050189/144943086-8e41fe4b-23fa-45d9-9375-51238b428187.png)

i got this error message after starting a strategy!

any tips?

**Comment by andresilvasantos** [comment]
Did you solve your issue? If you're using a proxy / vpn, please check if [this helps](https://stackoverflow.com/questions/53593182/client-network-socket-disconnected-before-secure-tls-connection-was-established).

**Comment by Zack00e** [comment]
Unfortunately I did not fix it, no I'm not using VPN or proxy!

**Comment by andresilvasantos** [comment]
Try to open [this link](https://api.binance.com/) in your browser and check if the window title changes to "Test OK".

**Comment by Zack00e** [comment]
yes, it shows "Test OK" i don't know what caused the issue! Maybe from script?
